bin/stress-test/src/main.rs

```diff
@@ -0,0 +1,117 @@
+use anyhow::Context;
+use miden_lib::{
+    accounts::{auth::RpoFalcon512, faucets::BasicFungibleFaucet, wallets::BasicWallet},
+    transaction::TransactionKernel,
+};
+use miden_node_block_producer::{
+    batch_builder::TransactionBatch, config::BlockProducerConfig, server::BlockProducer,
+    test_utils::MockProvenTxBuilder,
+};
+use miden_node_store::{config::StoreConfig, server::Store};
+use miden_objects::{
+    accounts::{AccountBuilder, AccountIdAnchor, AccountStorageMode, AccountType},
+    assets::{Asset, FungibleAsset, TokenSymbol},
+    crypto::dsa::rpo_falcon512::SecretKey,
+    testing::notes::NoteBuilder,
+    transaction::OutputNote,
+    Digest, Felt,
+};
+use miden_processor::crypto::RpoRandomCoin;
+use rand::Rng;
+use tokio::task::JoinSet;
+
+#[tokio::main]
+async fn main() {
+    let block_producer_config = BlockProducerConfig::default();
+    let store_config = StoreConfig::default();
+    let mut join_set = JoinSet::new();
+
+    // Start store
+    let store = Store::init(store_config).await.context("Loading store").unwrap();
+    let _ = join_set.spawn(async move { store.serve().await.context("Serving store") }).id();
+
+    // Start block-producer
+    // TODO: is the full the BlockProducer needed? we should instantiate only a BlockBuilder
+    let block_producer = BlockProducer::init(block_producer_config)
+        .await
+        .context("Loading block-producer")
+        .unwrap();
```

Comment:
I was actually thinking that we'd be constructing `Block` struct directly (we might need to do some extra bookkeeping to do this, specifically to keep data structures for updating account and nullifier tree roots) and that should be much faster than going through the block producer.